### PR TITLE
Compatibility with Rails 4.2

### PR DESCRIPTION
**Pull request by pschambacher**
:bullettrain_side::arrow_upper_right: 

/cc @zendesk/zendesk-rails-upgraders 

### Risks
 - None

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ rvm:
 script : bundle exec rake test
 bundler_args: ""
 gemfile:
-  - gemfiles/rails32.gemfile
-  - gemfiles/rails40.gemfile
-  - gemfiles/rails41.gemfile
+  - gemfiles/rails3.2.gemfile
```
Comment (grosser) [inline .travis.yml]: sweet consistency :)

File: test/logged_smtp_delivery_test.rb [hunk 2]
```diff
@@ -1,5 +1,11 @@
 require_relative 'helper'
 
+DELIVER_METHOD = if ActionMailer::VERSION::MAJOR > 4 || (ActionMailer::VERSION::MAJOR == 4 && ActionMailer::VERSION::MINOR >= 2)
```
Comment (grosser) [inline test/logged_smtp_delivery_test.rb]: A tad more readable: `ActionMailer::VERSION::STRING >= "4.2.0"`

**Comment by grosser** [comment]
:+1:

**Comment by pschambacher** [comment]
@grosser I'll leave the rake release to you as I don't have credentials :smile:

**Comment by grosser** [comment]
what's your rubygems email address ?
also your website is down http://www.pierreschambacher.com/

**Comment by grosser** [comment]
released 2.0.3

**Comment by pschambacher** [comment]
@grosser https://rubygems.org/profiles/PSchambacher email is my zendesk one

Regarding my website I stopped paying for it, 2 years without an update wasn't great :sweat_smile:

**Comment by grosser** [comment]
then link to you twitter or something :)

added you to the gem!

**Pull request merged**